generics/helpers/organisations.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * name : organisations.js
+ * author : Rakesh kumar
+ * Date : 26-Aug-2020
+ * Description : All organisations related functions.
+ */
+
+ //Dependinces
+ const sunbirdService =
+    require(GENERIC_SERVICES_PATH + "/sunbird");
+const sessionHelpers = require(GENERIC_HELPERS_PATH + "/sessions");
+
+/**
+  * To get organisations list
+  * @method
+  * @name getOrganisationlist
+  * @param { object } userProfileInfo - user profile information
+  * @param  {String} userId  - userId
+  * @param  {String} userToken  - user access token
+  * @param {Array} roles - roles of users
+  * @returns {boolean} return boolen value
+  * */
+
+ function getOrganisationlist(token, userId = "",roles=[]) {
```

Comment:
@rakeshSgr - Move this inside existing modules and don't do a direct query on models of other modules.